modules/apps/staging/staging-bar-web/src/main/resources/META-INF/resources/view_layout_revision_details.jsp

```diff
@@ -149,144 +166,32 @@ else {
 		onlyActions="<%= true %>"
 	/>
 
-	<li class="control-menu-nav-item">
-		<div class="d-none d-sm-block dropdown">
-			<button aria-haspopup="true" class="component-action dropdown-toggle taglib-icon" data-toggle="liferay-dropdown">
-				<aui:icon cssClass="<%= StringPool.BLANK %>" image="ellipsis-v" markupView="lexicon" />
-
-				<span class="sr-only">
-					<liferay-ui:message key="options" />
-				</span>
-			</button>
-
-			<ul class="dropdown-menu dropdown-menu-right" role="menu">
-				<li>
-					<a class="dropdown-item" href="javascript:void(0);" id="manageLayoutSetRevisions" onclick="<portlet:namespace />openSitePagesVariationsDialog();">
```

Comment:
Hi @BarbaraCabrera , could you please keep this IDs, as they are reference points for Functional tests / Poshi tests?